.github/workflows/bump-cli.yml

```
@@ -0,0 +1,32 @@
+name: Bump CLI version
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+  schedule:
```

Actually, let's make sure this runs when this particular file changes.

```suggestion
  pull:
    paths:
      - .github/actions/create-pr/action.yml
  schedule:
```